modules/oe_whitelabel_agenda/oe_whitelabel_agenda.module

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/**
+ * @file
+ * OE Whitelabel Agenda module.
+ */
+
+declare(strict_types=1);
+
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Render\Element;
+use Drupal\Core\Render\RendererInterface;
+use Drupal\link_description\Plugin\Field\FieldType\LinkDescriptionItem;
+use Drupal\oe_agenda\Entity\DayInterface;
+use Drupal\oe_agenda\Entity\SessionInterface;
+
+/**
+ * Implements hook_preprocess_oe_agenda() for oe_default agenda.
+ */
+function oe_whitelabel_agenda_preprocess_oe_agenda__oe_default(array &$variables): void {
+  foreach (Element::children($variables['content']['oe_agenda_days']) as $index) {
+    $day = $variables['content']['oe_agenda_days'][$index]['#oe_agenda_day'];
+    assert($day instanceof DayInterface);
+    // Pass agenda days as accordion items to the template.
+    $variables['days'][] = [
+      'title' => $day->get('oe_day_date')->first()->view('default'),
+      'content' => $variables['content']['oe_agenda_days'][$index],
+    ];
+  }
+}
+
+/**
+ * Implements hook_preprocess_oe_agenda_session() for oe_default session.
+ */
+function oe_whitelabel_agenda_preprocess_oe_agenda_session__oe_default(array &$variables): void {
+  $session = $variables['entity'];
+  assert($session instanceof SessionInterface);
+  $renderer = \Drupal::service('renderer');
+  assert($renderer instanceof RendererInterface);
+
+  // Build items for description list pattern displaying session links.
```

Comment:
Originally, I wanted to build the description list in the template, without a preprocess. I decided to do it in a preprocess because:
1. we need to add the external icon for external URLs
2. the description list output for the description text is using nl2br.. here I changed that to processed text so the new lines will be paragraphs, which I think is better formatting